packages/@coorpacademy-components/src/organism/mobile-intro-popin/test/fixtures/with-icon.ts

```diff
@@ -0,0 +1,21 @@
+import {NovaCompositionCoorpacademyInformationIcon as InformationIcon} from '@coorpacademy/nova-icons';
```

Comment:
not relevant to the PR but caught incomplete fixture warning for mobile coverage (that affected the diff on this PR - buggy CodeCov calculation)